stock_picking_analytic/views/stock_picking.xml

```diff
@@ -41,9 +56,9 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="stock.view_picking_internal_search" />
         <field name="arch" type="xml">
-            <field name="name" position="after">
+            <xpath expr="//field[@name='name']" position="after">
                 <field name="analytic_account_id" />
-            </field>
+            </xpath>
```

Comment:
This change seems to add no value, so it's better to undo it, if you don't mind.